Fitness/Views/Cells/DiscoverProsCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ProCell.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 4/22/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import Alamofire
+import AlamofireImage
+
+class DiscoverProsCell: UICollectionViewCell {
+    
+    // MARK: - INITIALIZATION
+    static let identifier = Identifiers.discoverProsCell
+    var collectionView: UICollectionView!
+    let itemSize = CGSize(width: 228.0, height: 195.0)
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        // COLLECTION VIEW LAYOUT
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal
+        layout.minimumInteritemSpacing = 16
+        layout.itemSize = itemSize
+        
+        collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        
+        collectionView.backgroundColor = .white
+        collectionView.showsHorizontalScrollIndicator = false
+        collectionView.delaysContentTouches = false
+        collectionView.accessibilityIdentifier = Identifiers.discoverProsCell
```

Comment:
the collectionView's dataSource and delegate should be this cell so you should have
```
collectionView.dataSource = self
collectionView.delegate = self
```